src/Engine/Renderer/Material/BlinnPhongMaterial.cpp

```diff
@@ -95,16 +93,16 @@ void BlinnPhongMaterial::bind( const ShaderProgram* shader ) {
 }
 
 bool BlinnPhongMaterial::isTransparent() const {
-    return ( m_alpha < 1.0 ) || ( m_kd[3] < 1.0 ) || Material::isTransparent();
+    return ( m_alpha < Scalar( 1 ) ) || ( m_kd[3] < Scalar( 1 ) ) || Material::isTransparent();
```

Comment:
Simply write `xxx < 1`.
C++ will always convert the operand to the good type (https://en.cppreference.com/w/cpp/language/operator_comparison : 

> If the operands has arithmetic or enumeration type (scoped or unscoped), usual arithmetic conversions are performed on both operands following the rules for arithmetic operators. The values are compared after conversions

Reply:
would `1_ra` be ok?